go.mod

```diff
@@ -4,11 +4,12 @@ go 1.13
 
 require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/emicklei/go-restful v2.11.1+incompatible // indirect
```

Comment:
Please, undo the changes to the go.mod and go.sum, as they aren't required for this PR.

Reply:
done